test/api.mocha.coffee

```diff
@@ -350,6 +356,11 @@ describe "API", ->
 
               # Send them invitations
               (_party, cb) ->
+                 # @TODO - _party should be array of newly
```

Comment:
Maybe it's meant to do that as a quick-and-dirty test to ensure that multiple user objects can be added to the party? From the point of view of actually adding users, it doesn't really matter if they're all the same user? Not ideal, but it might at least prove that arrays work. :)